packages/@okta/i18n/src/properties/login.properties

```diff
@@ -992,6 +992,8 @@ authfactor.challenge.suspended_factor = This factor is suspended for your accoun
 oie.post.password.update.auth.failure.error = Your password has been updated but there was a problem signing you in. Please try again or contact your administrator.
 ## {0} is the name of the application
 oie.assurance.unsatisfiable = Unable to login to "{0}" at this time. For assistance, please contact your administrator.
+errors.E0000150 = You have reached the limit of sms requests, please try again later.
+errors.E0000151 = You have reached the limit of call requests, please try again later.
```

Comment:
Can these please be moved to the [other `errors.*`](https://github.com/okta/okta-signin-widget/blob/3d2205e8e441a71cadd950077cf9c880e2515028/packages/%40okta/i18n/src/properties/login.properties#L174) properties?

Reply:
They are OIE server-side errors and hence thought this would be a better home for them. I prefer to merge this PR now since we're close to cut-off.
Please let me know if you want them moved and I'll open another PR later. Thank you